lib/runner/variant-reassignment.js

```diff
@@ -10,14 +10,16 @@ import log from '../services/logger'
 export default class VariantReassignment {
 
   constructor (client, logger, errorCallback = this._defaultErrorCallback.bind(this),
-               retainExistingData = []) {
+               retainExistingData = [],
+               whiteListedLocales = []) {
```

Comment:
Better use the name `allowedLocales` 😉

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/88/commits/ba7dbe35167527da1bc2d1ad5e4bebd0614f1e74